File: JAVA/PROGRAM/Ram.java
public class Ram extends Komponen {
    private int kapasitasGb;
    private String ddr;
    
    // Konstruktor
    public Ram() {
        super();
        this.kapasitasGb = 0;
        this.ddr = "";
    }
    
    public Ram(int kapasitasGb, String ddr, String merk, String nama) {
        super(merk, nama);
        this.kapasitasGb = kapasitasGb;
        this.ddr = ddr;
    }
    
    // Getter dan Setter
    public int getKapasitasGb() {
        return kapasitasGb;
    }
    
    public void setKapasitasGb(int kapasitasGb) {
        this.kapasitasGb = kapasitasGb;
    }
    
    public String getDdr() {
        return ddr;
    }
    
    public void setDdr(String ddr) {
        this.ddr = ddr;
    }
}

File: PYTHON/PROGRAM/Ram.py
from Komponen import Komponen

class Ram(Komponen):
    def __init__(self, kapasitas_gb=0, ddr="", merk="", nama=""):
        super().__init__(merk, nama)
        self.__kapasitas_gb = kapasitas_gb
        self.__ddr = ddr
    
    # Getter
    def get_kapasitas_gb(self):
        return self.__kapasitas_gb
    
    def get_ddr(self):
        return self.__ddr
    
    # Setter
    def set_kapasitas_gb(self, kapasitas_gb):
        self.__kapasitas_gb = kapasitas_gb
    
    def set_ddr(self, ddr):
        self.__ddr = ddr